pkg/alertmanager/api_test.go

```diff
@@ -633,7 +633,7 @@ func TestMultitenantAlertmanager_DeleteUserConfig(t *testing.T) {
 
 func TestAMConfigListUserConfig(t *testing.T) {
 	testCases := map[string]*UserConfig{
-		"user1": {
+		"user1|user2|user3": {
```

Comment:
Suggest adding this as an additional case, not replacing the single-tenant case.